Mathlib/Topology/Sets/Opens.lean

```diff
@@ -306,6 +306,12 @@ theorem IsBasis.isCompact_open_iff_eq_finite_iUnion {ι : Type*} (b : ι → Ope
     simp
   · exact hb'
 
+lemma IsBasis.le_iff {α} {t₁ t₂ : TopologicalSpace α}
+    {Us : Set (Opens α)} (hUs : IsBasis Us) : -- basis for t₂
```

Comment:
What in the syntax makes us sure that it is a basis for `t₂`? Wouldn't it be safer to enter `@IsBasis α t₂ Us`?